### ISSUE TITLE
something awful happens with new versions of ncurses

### ISSUE THREAD

**Issue by jwise** [jwise/naim#25]
naim _really_ doesn't get along well with new versions of ncurses (maybe?  new Debians?  new screens?), and goes spewing crap everywhere, I think when it tried to update the window title.  And, for bonus points, it hangs mrxvt whenever it tries to do this outside screen!

**Comment by jwise** [comment]
nopping out nw_titlef seems to help

**Comment by georgedorn** [comment]
Yeah, noticed this when I upgraded to Ubuntu 14.04 LTS.  I'm apparently running 5.9+20140118-1ubuntu1.
